flask_monitoringdashboard/main.py

```diff
@@ -48,8 +49,8 @@ def endpoint():
     # if session_scope is imported at the top of the file, the database config won't take effect
     from flask_monitoringdashboard.database import session_scope
 
-    with session_scope() as db_session:
-        print(db_session.bind.dialect.name)
+    with session_scope() as session:
+        print(session.bind.dialect.name)
 
     print("Hello, world")
```

Comment:
not sure this is useful for anything.

Reply:
Cool idea Johan, I'll create an examples folder in another pull request.